src/lang.ts

```diff
@@ -39,7 +40,7 @@ const lang: LangInterface = {
     /**
      * Stores the fallback messages
      */
-    fallbackMessages: require('./locales/en.js').default,
+    fallbackMessages: fallbackMessages,
```

Comment:
I think it would be better to do the following, **fallbackMessages: locales.en**, and do the same on line 87. This way there will be no need anymore to require the en.js